core/context/src/main/java/com/milesight/beaveriot/context/security/SecurityContextUserAspect.java

```diff
@@ -0,0 +1,54 @@
+package com.milesight.beaveriot.context.security;
+
+import com.milesight.beaveriot.context.util.AnnotationSpelExpressionUtil;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.After;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Before;
+import org.aspectj.lang.annotation.Pointcut;
+import org.aspectj.lang.reflect.MethodSignature;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.context.annotation.EnableAspectJAutoProxy;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+
+/**
+ * @author loong
+ * @date 2024/12/5 13:00
+ */
+@Component
```

Comment:
It is recommended to use @Configuration annotation to assemble spring beans for easy management

Reply:
Use @ Component because its responsibilities are relatively single, which reduces the complexity of configuration or makes it difficult to manage separately